contracts/ExpectedRate.sol

```diff
@@ -12,7 +12,6 @@ contract ExpectedRate is Withdrawable, ExpectedRateInterface, Utils2 {
     KyberNetwork public kyberNetwork;
     uint public quantityFactor = 2;
     uint public worstCaseRateFactorInBps = 50;
-    uint constant UNIT_QTY_FOR_FEE_BURNER = 10 ** 18;
```

Comment:
should be removed in another PR
right?
please pull development to align files.